jetbrains-core/src/software/aws/toolkits/jetbrains/ui/wizard/SamInitSelectionPanel.java

```diff
@@ -44,6 +44,8 @@
                                .sorted()
                                .forEach(y -> runtime.addItem(y));
 
+        runtime.setSelectedItem(this.projectSettings.runtime);
```

Comment:
I am not sure looking at the existing code if this is actually safe since I only see it set the first time in the runtimeUpdate() call. It seems like it show be throwing an NPE due to the property is lateinit.